### ISSUE TITLE
failed to import in python 3.6

### ISSUE THREAD

**Issue by mh739025250** [polm/fugashi#11]
When I import fugashi, it will raise an error like this:

```
from fugashi import GenericTagger
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "fugashi/fugashi.pyx", line 9, in init fugashi
TypeError: namedtuple() got an unexpected keyword argument 'defaults'
```

My python version is 3.6.7, and fugashi version is 0.1.8.

The error is caused by nametuple doesn't have an 'defaults' argument in python 3.6. The 'defaults' argument is added in python 3.7.

I have searched in stackoverflow, and a fix can be like this:

```
from collections import namedtuple
Node = namedtuple('Node', 'val left right')
Node.__new__.__defaults__ = (None,) * len(Node._fields)
```

Or you can make fugashi requires python version >=3.7.

**Comment by polm** [comment]
Thanks for the report! Since it looks like 3.6 is only receiving security fixes at this point I'll make fugashi require >=3.7.

**Comment by polm** [comment]
I just released 0.1.9, which incorporates this change. Thanks again for the report!
